core/io/config_file.cpp

```diff
@@ -101,18 +75,33 @@ bool ConfigFile::has_section_key(const String &p_section, const String &p_key) c
 	return values[p_section].has(p_key);
 }
 
-void ConfigFile::get_sections(List<String> *r_sections) const {
+Vector<String> ConfigFile::get_sections() const {
+	Vector<String> r_sections;
```

Comment:
```suggestion
	Vector<String> sections;
```
`r_` prefix is reserved for method's parameters.
Same below.